storage.go

```diff
@@ -590,6 +590,54 @@ func (storage DBStorage) StoreTableMetadataIntoFile(fileName string, tableNames
 	return nil
 }
 
+// StoreTableMetadataIntoS3 method stores metadata about given tables into
+// S3 or Minio.
+func (storage DBStorage) StoreTableMetadataIntoS3(ctx context.Context,
+	minioClient *minio.Client, bucketName string, objectName string,
+	tableNames []TableName) error {
+
+	buffer := new(bytes.Buffer)
+
+	// initialize CSV writer
+	writer := csv.NewWriter(buffer)
+
+	err := writer.Write([]string{"Table name", "Records"})
```

Comment:
This code (until line 616) is duplicated from the previous function.

Reply:
it is

Reply:
updated